experiments/functional/backup_and_restore/induce_failure.yml

```diff
@@ -0,0 +1,26 @@
+---
+- block:
+   
+   - name: Getting the volume name
+     shell: kubectl get pvc {{ app_pvc }} -n {{ app_ns }} -o jsonpath='{.spec.volumeName}'
+     register: vol_name
+     
+   - name: Getting the CVR name from the corresponding PV
+     shell:  kubectl get cvr -n openebs -l openebs.io/persistent-volume={{ vol_name.stdout }} -o jsonpath='{range.items[*]}{.metadata.labels.cstorpool\.openebs\.io/name}{"\n"}{end}' | head -2
```

Comment:
why the top two csps are picked

Reply:
It should be like this.
1. Find CStorBackup CR for the given backup.
2. Find the relevant CVR for above CStorBackup CR. cstorbackup.labels[cstorpool.openebs.io/uid] == cvr.labels[cstorpool.openebs.io/uid]
3. From CVR find pool pod. cvr.labels[cstorpool.openebs.io/name]. kubectl get pods -n openebs  -o jsonpath='{.items[?(@.metadata.labels.openebs\.io/cstor-pool=="cstor-pool-6lod")].metadata.name}'

5. Restart the pod.